ui/raidboss/data/06-ew/ultimate/the_omega_protocol.txt

```diff
@@ -207,15 +209,19 @@ hideall "--sync--"
 789.0 "Hello, Near World 3" #sync / 1[56]:[^:]*:Omega-M:7B8A:/
 
 # Sigma Version
-# TODO: Adjust targetable/untargetable
-791.0 "--targetable--"
+# TODO: Account for patterns
```

Comment:
What does this mean?

Reply:
Ah looking back at it, I guess I already had the Staff vs Blades patterns. Was not sure if clockwise vs counterclockwise existed and had different timings.

For Omega, I thought there are 4 patterns like with P2. Wasn't sure if which spells happen. I think the timings are the same though.

I'll remove the comments.